custom/icds_reports/utils/aggregation_helpers/distributed/awc_location.py

```diff
@@ -79,7 +79,7 @@ def generate_csv(self):
                 })
                 if loc_type == 'supervisor':
                     loc.update({
-                        'supervisor_deprecates': ','.join(metadata.get('deprecates', [])),
+                        'supervisor_deprecates': ','.join(metadata.get('deprecates')),
```

Comment:
i think it should actually just be `metadata.get('deprecates')` if it doesnt exist null is fine, but the join is not correct since this is already a comma separated list now